src/files.js

```diff
@@ -24,551 +24,553 @@
 if ( typeof window !== 'undefined' ) global = window;
 
 FOAM_FILES([
```

Comment:
This is a huge diff.  Can you look at disabling your JS formatter?  Until FOAM decides upon a standard code formatter to use better to just let code live as is.

In fact the style guide the FOAM uses recommends leaving style changing changes to their own PRs.

https://google.github.io/styleguide/jsguide.html#policies-reformatting-existing-code

Reply:
Fixed